packages/wallets/scripts/update-translations.sh

```diff
@@ -41,10 +41,13 @@ if [ "$NODE_ENV" = "staging" ]; then
         fi
     fi
 
-    echo "Running commands for staging environment..."
-    message "Uploading source file to Crowdin" &&
-    retry crowdin upload sources --auto-update &&
-    message "Complete, new translations have been uploaded to Crowdin" &&
+    GENERATE_KEY=src/utils/generate-keys.ts
+    if [ -f "$GENERATE_KEY" ]; then
+      message "Uploading source file to Crowdin" &&
+      retry crowdin upload sources --auto-update &&
+      message "Complete, new translations have been uploaded to Crowdin"
```

Comment:
Just a note in case anyone is wondering, other pull requests without the newly generated translation keys changes will upload the generated `messages.json` with the old generated keys. This causes issues in staging where i18n can't find the translated values since the keys are based on the newly generated keys (SHA256) and replaces the values with the keys itself.
So this is a failsafe to ensure old pull requests without the new generated keys function in `packages/wallets/src/utils/generate-keys.ts` will not upload their generated `messages.json` in wallets